pom.xml

```diff
@@ -1,17 +1,146 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>net.conjur.api</groupId>
+  <groupId>com.cyberark.conjur.api</groupId>
   <artifactId>conjur-api</artifactId>
-  <version>2.2.1</version>
+  <!-- Only bump version number,
+  SNAPSHOT is automatically removed during release deployments -->
+  <version>2.2.1-SNAPSHOT</version>
```

Comment:
This PR change requires a bump in version now as well

Reply:
I think we should do that as a follow-up PR. @izgeri ?